antd/antd-theme.js

```diff
@@ -1,4 +1,4 @@
-import { COLORS } from '../src/theme/theme.default'
+const { COLORS } = require('../src/theme/theme.default')
```

Comment:
Because node is using CommonJS syntax by default I decided to introduce some improvements in the `webpack` configs. Originally, the `webpack` config mixed CommonJS and ESM syntax. That issue was worked around by running `webpack` with `node -r esm`. I prefer to use a direct solution if possible. So I converted files that are used in `webpack` configs to CommonJS syntax.